docs/docs/examples/custom-header.md

```diff
@@ -4,8 +4,70 @@ sidebar_position: 2
 
 # Custom Header
 
-Check out a blogpost explaining in details how to customize Avatar Header:
+To create custom header layout, you'll have to use `StickyHeader(ScrollView|FlatList|SectionList)` & `useStickyHeaderScrollProps`. If you want to use custom scroll component, instead of `StickyHeader(ScrollView|FlatList|SectionList)`, you can wrap your custom scroll component in `withStickyHeader` HOC.
 
-[Implement an App Store Header in React Native With Sticky Parallax Library](https://www.netguru.com/codestories/implement-an-app-store-header-in-react-native-with-sticky-parallax-library)
+## Scroll props
 
-<iframe title="vimeo-player" src="https://player.vimeo.com/video/544879484" width="320" height="693" frameborder="0" allowfullscreen></iframe>
+`useStickyHeaderScrollProps` is a hook responsible for creating "snap effect" behavior
+
+```js
```

Comment:
Previously you've used `.tsx` syntax extension. What do you think about unifying snippets to ts/tsx?